src/views/listview/list-view-directive.js

```diff
@@ -184,6 +197,27 @@
           return $scope.showDisabled && (item.name === "John Smith");
         };
 
+        var dragEnd = function() {
+          $scope.eventText = 'drag end\r\n' + $scope.eventText;
+        };
+        var dragMoved = function() {
```

Comment:
Shouldn't this dragMoved functionality be the default behavior in the directive instead of leaving it up to the end users to implement it correctly?